src/leaflike/user/core.clj

```diff
@@ -0,0 +1,35 @@
+(ns leaflike.user.core
+  (:require [leaflike.user.db :as user-db]
+            [leaflike.user.auth :as auth]
+            [leaflike.user.validator :as validator]))
+
+(defn signup
+  [request]
+
+  (let [body  (clojure.walk/keywordize-keys (-> request :params))
+        valid (validator/valid-registration? body)]
```

Comment:
Let's use a more representative name than 'valid', 'valid' is generic and doesn't tell valid-of-what or valid-what